### ISSUE TITLE
Undo rating

### ISSUE THREAD

**Issue by Haoran-Hu** [open-learning-exchange/planet#6450]
After a user rates a resource or a course, a user can only change the rating instead of canceling the rating.

Is it the desired behavior?

**Comment by paulbert** [comment]
Yes.  At this point we want to make this as simple as possible so it's easy to use. Closing

**Comment by paulbert** [comment]
Reconsidering, and reopening for a larger discussion Friday.
